okhttp/src/main/java/okhttp3/RealCall.java

```diff
@@ -168,6 +173,8 @@ String redactedUrl() {
   }
 
   Response getResponseWithInterceptorChain() throws IOException {
+    retryAndFollowUpInterceptor.setEventListener(eventListener);
```

Comment:
should we put the eventListener on the RealInterceptorChain? That might make it easier for code that needs access to it to get access to it